.github/workflows/build-package.yml.template

```diff
@@ -67,7 +67,7 @@ jobs:
   upload-popular-package:
     name: Upload
     needs: build-popular-package
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-latest
```

Comment:
the general approach with building for linux is "pick the oldest version of glibc that you can get away with, so that it runs on as many machines as possible".

What was the error that caused you to bump the version?

Reply:
no error, just saw that it worked and it's good to stay up to date.

Reply:
In this case, updating our glibc version is an abi break (it means our packages won't run on older Linux distros with older versions of glibc), so lets only do it when canonical + GitHub stop supporting Ubuntu 20.04 for security patches.

Reply:
> so lets only do it when canonical + GitHub stop supporting Ubuntu 20.04 for security patches.

Actually zig-cc can be used to build against old glibc, or arbitary glibc version it supports.